locale/en/submission.po

```diff
@@ -2454,3 +2454,51 @@ msgstr "Please disclose any competing interests this author may have with the re
 
 msgid "submission.localeNotSupported"
 msgstr "The language of the submission ({$language}) is not one of the supported submission languages. You can still edit the submission details but new submissions are not currently accepted with this language."
+
+msgid "publication.changeVersionStage.title"
+msgstr "Version Stage"
+
+msgid "publication.changeVersionStage.buttonLabel"
+msgstr "Manage version stage"
+
+msgid "publication.versionStage.minorOrMajor"
+msgstr "Version Impact"
+
+msgid "publication.versionStage.minorOrMajor.description"
```

Comment:
This needs a translation, or to be removed if it's redundant. But I'd suggest not including the locale file changes -- they're likely to be made redundant by someone else's UI implementation.